Automattic/jetpack
# Update boost score bar to use package api

This updates the Boost plugin to get Boost Scores from the API package instead of the local code

## Proposed changes:
 * Remove functions relating to boost scores locally
 * Import boost-score-api `js-package` and use it instead

### Other information:


## Jetpack product discussion
P2: pc9hqz-1tV-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

1. Checkout this patch or get an ephemeral site with the Jetpack Beta plugin having Jetpack Boost go off this branch
2. If you haven't already, Install the Jetpack Boost plugin (can be free or paid)
3. Go to `/wp-admin/admin.php?page=jetpack-boost`
4. Make sure the Scores load as expected
![image](https://github.com/Automattic/jetpack/assets/65001528/891f591b-fb2e-4215-911a-27966175a971)
